### ISSUE TITLE
time too long

### ISSUE THREAD

**Issue by 15757170756** [JayYangSS/HazeRemove#1]
hello thanks for your code on hazeremove , I download it and run it. But i find it use too much time to get the result.it takes me about 1mins to get the result.

**Comment by JayYangSS** [comment]
Yeah, I haven't optimize the code. So I would be very happy if you are interested in optimize the code. :D@15757170756
